src/index.css

```diff
@@ -11,5 +12,5 @@ body {
 
 #root {
   height: 100%;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
Not sure about these changes, maybe let's look how it behaves in the development environment.

Reply:
the purpose of this change is to make sure that the table is controlling height and scroll,
this how it looks with current changes
![image](https://github.com/graasp/graasp-app-calculator/assets/49619087/b85952a2-f73c-462a-a186-222f20ecf2ca)

and this is how it would be with the previous version
![image](https://github.com/graasp/graasp-app-calculator/assets/49619087/46f36ab8-90f7-4994-8542-e2ab7a1f8e70)